### PR TITLE
python3-plotly: update to 5.22.0.

### PR DESCRIPTION
**Pull request by Vinfall**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, x86_64-glibc

I've tested using a few scripts I wrote and they work perfectly after tweaking to new syntax in the new version.

### CODE REVIEW

File: srcpkgs/python3-plotly/template [hunk 1]
```diff
@@ -11,7 +11,7 @@ license="MIT"
 homepage="https://plot.ly/python/"
 changelog="https://github.com/plotly/plotly.py/blob/master/CHANGELOG.md"
 distfiles="${PYPI_SITE}/p/plotly/plotly-${version}.tar.gz"
-checksum=822eabe53997d5ebf23c77e1d1fcbf3bb6aa745eb05d532afd4b6f9a2e2ab02f
+checksum=859fdadbd86b5770ae2466e542b761b247d1c6b49daed765b95bb8c7063e7469
 
 do_check() {
```
Comment (ahesford) [inline srcpkgs/python3-plotly/template:16]: Please migrate this to `make_check=no # no tests are shipped` after `checksum`.

**Review by ahesford — CHANGES_REQUESTED** [review]

File: srcpkgs/python3-plotly/template (hunk 1 above)
Comment (Vinfall) [reply srcpkgs/python3-plotly/template:16]: Should `do_check` function be kept untouched?

Comment (ahesford) [reply srcpkgs/python3-plotly/template:17]: No, you can remove the whole function.

Comment (Vinfall) [reply srcpkgs/python3-plotly/template]: Done

**Pull request merged**